# Single_Cell_RNA_sequncing_data_visualization

An data visualizaiton project for analysing single cell RNA sequencing data from Gambardella, Viscido et al. 2022.
In the project, the cancer subtpye specific gene transcription and distribution in single cell was analysed.


[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)



---

## Table of Contents

1. [About the Project](#about-the-project)
2. [Getting Started](#getting-started)
3. [Usage](#usage)
4. [Features](#features)
5. [Contributing](#contributing)
6. [License](#license)
7. [Acknowledgments](#acknowledgments)

---

## About the Project

Briefly explain:
- PCA reducing dimensions 
- Visulizing cancer cell amount in each sub cancer type.
- Transcription enrichment in different pathways.
- Gene transcription level in single cells and amount cancer subtypes.

---

## Getting Started

Step-by-step instructions to get a local copy running.

### Prerequisites

List tools and dependencies needed:
- Python 3.10+
- pip install scanpy
- PCA
- umap

### Installation

Provide clear installation steps:

```bash
pip install scanpy

# Navigate to the project directory
cd repo

# Install dependencies

```
### Load the file and Handling Missing values

### Data Visualization

## Contributing

Contributions are welcome! Please follow these steps:

1. Fork the repository.
2. Create your feature branch (`git checkout -b feature/new-feature`).
3. Commit your changes (`git commit -m 'Add new feature'`).
4. Push to the branch (`git push origin feature/new-feature`).
5. Open a pull request.

Refer to [CONTRIBUTING.md](CONTRIBUTING.md) for detailed guidelines.

---

## License

Distributed under the MIT License. See `LICENSE` for more information.

## Citation
dibbelab, & Gennaro Gambardella. (2022). dibbelab/singlecell_bcatlas: Single-cell BC Atlas (v1.0.0). Zenodo. https://doi.org/10.5281/zenodo.5939376

